Install fails

### URL:

node

### Browser Name:

all

### Browser Version:

latest main

### Operating System:

Mac

### How to reproduce the issue:

Installing the site after pulling main from over a month ago on this machine, I get the following.

Creating as triage to see if we need to add doc-kit.

```
➜  nodejs.org git:(main) pnpm i
Scope: all 6 workspace projects
Lockfile is up to date, resolution step is skipped
Packages: +1626
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 WARN  Tarball download average speed 42 KiB/s (size 192 KiB) is below 50 KiB/s: https://codeload.github.com/nodejs/doc-kit/tar.gz/be7fc307395005ea362d035c43e3818650bf075f (GET)
Packages are cloned from the content-addressable store to the virtual store.
  Content-addressable store is at: /Users/megan/Library/pnpm/store/v10
  Virtual store is at:             node_modules/.pnpm
 ERR_PNPM_GIT_DEP_PREPARE_NOT_ALLOWED  Failed to prepare git-hosted package fetched from "https://codeload.github.com/nodejs/doc-kit/tar.gz/be7fc307395005ea362d035c43e3818650bf075f": The git-hosted package "@nodejs/doc-kit@" needs to execute build scripts but is not in the "onlyBuiltDependencies" allowlist.

Add the package to "onlyBuiltDependencies" in your project's pnpm-workspace.yaml to allow it to run scripts.
````

--- a/pnpm-workspace.yaml
+++ b/pnpm-workspace.yaml
@@ -12,6 +12,7 @@ catalog:
   typescript: 5.9.3
 
 onlyBuiltDependencies:
+  - '@nodejs/doc-kit'
   - '@swc/core'
   - '@tailwindcss/oxide'
   - '@vercel/speed-insights'

